.devcontainer/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10
+WORKDIR /scvi-tools
+COPY . .
+
+RUN pip install --upgrade pip
+RUN pip install "https://storage.googleapis.com/jax-releases/nocuda/jaxlib-0.4.1-cp310-cp310-manylinux2014_x86_64.whl"
```

Comment:
can you install mambaforge, mamba install scvi tools, then pip editable install?